src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -27,7 +28,13 @@ public static void printLongestWordTop100() throws IOException {
                 .get("src/main/resources/fp/war-and-peace.txt")), StandardCharsets.UTF_8);
         List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
 
-        // TODO 이 부분에 구현한다.
+        words.stream()
+                .filter(x -> x.length() > 12)
```

Comment:
stream API를 사용할 때 변수명을 의미있게 작성하면 더 가독성이 좋아질 것 같습니다. 
```suggestion
                .filter(word -> word.length() > 12)
```

Reply:
step2 반영하면서 수정하겠습니다!